draft-ietf-rats-architecture.md

```diff
@@ -1297,15 +1302,15 @@ The only entity that requires access to a source of time is the entity taking on
 Handles are different from nonces as they can be used more than once and can be used by more than one entity at the same time.
 Handles are different from timestamps as they do not have to convey information about a point in time, but their reception creates that information.
 The reception of a handle is similar to the event that increments a relative tickcounter.
-Receipt of a new handle invalidates a previously received handle. 
+Receipt of a new handle invalidates a previously received handle.
 
 In this example, Evidence generation based on received handles always uses the current (most recent) handle.
 As handles are distributed over the network, all involved entities receive a fresh handle at roughly the same time.
-The time that represents this event is named Handle received: time(HR). Due to distribution over the network, there is some jitter with respect to time(HR) for each involved entity.
-To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR') + offset - time(HR)`. The offset is typically based on a network's round trip time.
-Analogously, the generation of valid Evidence is only possible, if the age of the handle used is lower than the validity-duration: `time(HR) - time(EG) < validity-duration`.
+The time that represents this event is named Handle received: time(HR_x). Due to distribution over the network, there is some jitter with respect to time(HR_x) for each involved entity.
+To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR_v') + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
```

Comment:
```suggestion
To compensate for this jitter, there is a small period of overlap (a specified offset) in which both a current handle and corresponding former handle are valid in Evidence appraisal: `validity-duration = time(HR'_v) + offset - time(HR_v)`. The offset is typically based on a network's round trip time.
```
The HR should be prime'ed rather than the subscript